subprojects/core-api/src/main/java/org/gradle/api/toolchain/management/ToolchainManagement.java

```diff
@@ -24,7 +24,7 @@
  * currently existing toolchains (Java) and future ones.
  * <p>
  * One option available at present is using the <code>jvm-toolchains</code> plugin, which
- * adds a <code>jdks</code> block to it. This block allows for specifying which
+ * adds a <code>jvm</code> block to it. This block allows for specifying which
```

Comment:
What's the rationale behind this change? This sounds weird, as a JVM is a runtime, when a JDK is an SDK, which is closer to me to the notion of a toolchain.

Reply:
I would say JDK is too restrictive. One of the tools we get from toolchains is the launcher, which can come from a JRE. But all the tools are JVM related. So makes sense, we're just used to talk about JDKs in this context, I guess.

Reply:
Actually the better reason is to consider future Groovy, Scala and other JVM based languages/toolchains.

Reply:
I would argue that `javac` is _not_ a JVM, nor does it start a JVM, so it doesn't really make sense to me. I'm still a bit confused.